db/migrate/20160311231152_add_commit_sha_to_miq_ae_namespaces.rb

```diff
@@ -0,0 +1,11 @@
+class AddCommitShaToMiqAeNamespaces < ActiveRecord::Migration[5.0]
+  def change
+    add_column :miq_ae_namespaces, :commit_sha, :string
+    add_column :miq_ae_namespaces, :commit_time, :timestamp
+    add_column :miq_ae_namespaces, :commit_message, :text
+    add_column :miq_ae_namespaces, :git_repository_id, :bigint
+    add_column :miq_ae_namespaces, :branch, :string
+    add_column :miq_ae_namespaces, :tag, :string
```

Comment:
Also here...do you need both branch and tag?  Why the separation if you can only have one?